server/api/entryRouters.ts

```diff
@@ -0,0 +1,46 @@
+import express, { Request, Response, NextFunction } from "express";
+import { Entry } from "../db/index.js";
+import { EntryAttributes } from "../db/models/Entry.model.js";
+const router = express.Router();
+
+router.get("/", async (req: Request, res: Response, next: NextFunction) : Promise<void>=> {
+  try {
+    const entries: EntryAttributes[] = await Entry.findAll();
+    res.send(entries);
+  } catch (err) {
+    res.sendStatus(404);
+    next(err);
+  }
+});
+
+router.post("/", async (req: Request, res: Response, next: NextFunction): Promise<void>=> {
+    try {
+      const {entryType, date, creditDebit, amount, title, note, frequency} : EntryAttributes = req.body
+      // const createEntry= 
+      await Entry.create({
+        entryType, date, creditDebit, amount, title, note, frequency
+      })
+      res.sendStatus(204)
```

Comment:
We'll need this returned with the information from the database for use in redux. To make UUID of the entry match up. May also need some validation for it to be unique in some way? Essentially prevent hard duplicates of the exact same entry?